join.spec.md

```diff
@@ -280,25 +271,26 @@ In order to acquire a selection set on a entity within a subgraph, routers may i
 ```graphql example -- Portal query
 query {
   _entities(representations: $representations) {
-    # ...selections...
+    # ...selections
   }
 }
 ```
 
-`representations` MUST be a list of input objects, each of which consist of the `__typename` of the object type or interface along with the result of fetching one that subgraph's `key` fragment for the type.
+`representations` MUST be a list of input objects, each of which consist of the `__typename` of the object type or interface along with the result of fetching that subgraph's `key` for the type.
 
 ```graphql example -- Portal query with representations
 query {
   _entities(representations: [
     { __typename: "Image", url: "http://example.com/1.png"},
     { __typename: "Image", url: "http://example.com/2.png"},
   ]) {
-    # ...selection on Image...
+    # ... on Image { ... }
   }
 }
 ```
 
-The `_entities` resolver need not be explicitly defined in the subgraph schema. If it is defined, its specified types are ignored—it is assumed that the resolver is capable of resolving objects and interfaces from any key specified within the join schema.
+// TODO: is this correct? How can this possibly work without definining `_entities` resolver?
+The `_entities` resolver need not be explicitly be defined in the subgraph schema. If it is defined, its specified types are ignored. It will be assumed that the resolver is capable of resolving objects and interfaces from any key specified within the supergraph.
 
 # Validations
```

Comment:
We should remove this section and any links to it (eg, a link from "What this document isn't"), since they don't exist yet.

Reply:
Fixed on my branch